lib/OpenQA/Schema.pm

```diff
@@ -70,15 +72,17 @@ sub disconnect_db {
 
 sub dsn {
     my $self = shift;
-    $self->storage->connect_info->[0]->{dsn};
+    return $self->storage->connect_info->[0]->{dsn};
 }
 
 sub deployment_check {
+
```

Comment:
I don't care much be I'd prefer not having this extra blank line.

Reply:
I like blank lines before comment blocks. But it definitely is a matter of taste.

Reply:
I like how you guys argue about beautiful perl code as if such a thing existed :)

![](https://media.giphy.com/media/TA2qnvTHsUSSQ/giphy.gif)

Reply:
BTW: I *love* that movie - if you haven't seen 'The Help', fix that.

Reply:
I find the blank line makes the logic parts of the code more obvious as well, that is I wouldn't say before "comments" but rather comments introducing code blocks

Reply:
I said I don't care much :-)